routes/_components/dialog/components/ImageDialog.html

```diff
@@ -29,6 +29,9 @@
     max-width: calc(100vw - 20px);
     max-height: calc(100% - 20px);
     overflow: hidden;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
```

Comment:
I think these `background-repeat`/`-position`/`-size` etc. should probably be added to the `AutoplayVideo` itself, since that's where the `background-image` is set. it just keeps all the background logic in one place